packages/website/docs/SingleSelect.md

```diff
@@ -1,24 +1,46 @@
-## Guidelines
-
-### General
-
--   If space allows, and there are no more than 7 options to choose from, consider using [radio buttons](#/components/Radio) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
-
-### Options Sorting
-
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
-
-### Label
-
--   In most cases, you should include a label to identify the component.
-
+[Description]: # 
+[Single-select dropdown allow to select only one option from a predefined list. It is typically used when there are a large number of available options.]: # 
+
+# Best Practices 
+Use multi-select to allow user to select one option from a **long list that can be filtered**. Single-selects are more appropriate when the available list of options is very long or when the space is limited. 
```

Comment:
I'm not sure about talking about the multi select in the first sentence of the SingleSelect guidelines :see_no_evil: 

```suggestion
Single-selects are appropriate when the available list of options is very long or when the space is limited. 
To allow users to select one option from a **long list that can be filtered** use the multi-select . 
```

or something like that? :thinking:

Reply:
It's a coquille. It should have said Multi-selects. 😬